data/actionsmap/yunohost.yml

```diff
@@ -1178,6 +1178,33 @@ service:
                     nargs: "+"
                     metavar: NAME
 
+        ### service_reload()
+        reload:
+            action_help: Reload one or more services
+            arguments:
+                names:
+                    help: Service name to reload
+                    nargs: "+"
+                    metavar: NAME
+
+        ### service_restart()
+        restart:
+            action_help: Restart one or more services. If the services are not running yet, they will be started.
+            arguments:
+                names:
+                    help: Service name to restart
+                    nargs: "+"
+                    metavar: NAME
+
+        ### service_reload_or_restart()
+        reload_or_restart:
```

Comment:
Can you elaborate on the intention behind this `reload_or_restart` ? That's not such a big deal I guess, but I'm just scared by this considering that for example for nginx, doing a `restart` instead of `reload` has serious implications as in "this will disconnect every ongoing session".

I don't really understand the use case behind it. Is it just to copy-paste some systemd api ?

Reply:
I'm not an expert in systemd and I copied pasted the api.

But the use case of this command is if you are unsure whether the service has the functionality to reload its configuration, you can use the `reload-or-restart` command. This will reload the configuration in-place if available. Otherwise, it will restart the service.

If you use this on nginx, as nginx can be reloaded, it should not be restarted.